test-loop-tests/src/tests/spice.rs

```diff
@@ -92,20 +93,8 @@ fn test_spice_chain() {
             &epoch_manager.get_epoch_chunk_producers_for_shard(&epoch_id, shard_id).unwrap()[0];
 
         let view_client_handle = get_node_data(&node_datas, cp).view_client_sender.actor_handle();
-
         let view_client = test_loop_data.get_mut(&view_client_handle);
-
-        let query_response = view_client.handle(Query::new(
-            BlockReference::Finality(near_primitives::types::Finality::Final),
-            QueryRequest::ViewAccount { account_id: account.clone() },
-        ));
-
-        let query_response = query_response.unwrap();
-
-        let QueryResponseKind::ViewAccount(view_account_result) = query_response.kind else {
-            panic!();
-        };
-        view_account_result.amount
+        query_view_account(view_client, account.clone()).amount
```

Comment:
This callsite used to be Finality::Final but now it's Finality::None.
Just highlighting it, maybe it is not a relevant detail.